src/bindings/js/node/include/model_wrap.hpp

```diff
@@ -109,6 +109,13 @@ class ModelWrap : public Napi::ObjectWrap<ModelWrap> {
      */
     Napi::Value get_output_shape(const Napi::CallbackInfo& info);
 
+    /**
+     * @brief Helper function to access model output elements types.
+     * @return Napi::Value wrapping a TypeWrap object representing the element type of the requested output.
```

Comment:
There is no such thing as TypeWrap. 
```suggestion
     * @return Napi::String representing the element type of the requested output.
```